manifests/base/config/argocd-cm.yaml

```diff
@@ -21,6 +21,10 @@ data:
       - '.metadata.annotations."argocd.argoproj.io/refresh"'
       - '.metadata.annotations."argocd.argoproj.io/hydrate"'
       - '.operation'
+  resource.customizations.ignoreDifferences.argoproj.io_Application: |
```

Comment:
If these are required configuration for argo to work properly that should always be configured, they should be hardcoded in https://github.com/argoproj/argo-cd/blob/1c5d7f1f65909631d2c91fa791bac5269f56f5bb/util/settings/settings.go#L963